lib/locale_pack/railtie.rb

```diff
@@ -18,7 +18,7 @@ class Railtie < Rails::Railtie
         config.config_path = app.config.locale_pack[:config_path] if app.config.locale_pack.key?(:config_path)
         config.locale_path = app.config.locale_pack[:locale_path] if app.config.locale_pack.key?(:locale_path)
         config.output_path = app.config.locale_pack[:output_path] if app.config.locale_pack.key?(:output_path)
-        config.export_locales = app.config.i18n.available_locales if app.config&.i18n&.available_locales
+        config.export_locales = export_locales(app)
```

Comment:
This overrides the default `[]` even if the `i18n` config isn't available.
```suggestion
        locales = export_locales(app)
        config.export_locales = locales unless locales.nil?
```